rest-client-base/src/main/java/com/wultra/core/rest/client/base/DefaultRestClient.java

```diff
@@ -193,6 +200,12 @@ private void initializeWebClient() throws RestClientException {
         webClient = builder.baseUrl(config.getBaseUrl()).clientConnector(connector).build();
     }
 
+    private static void validateConfiguration(final RestClientConfiguration config) throws RestClientException {
+        if (config.isHttpBasicAuthEnabled() && config.isHttpDigestAuthEnabled()) {
```

Comment:
Just a hand-brake to have reasonable configuration and not try to configure both.

Reply:
Good idea, I just want to add comment about this, but it is already done. 👍